src/iocore/net/P_Connection.h

```diff
@@ -58,20 +58,8 @@ struct NetVCOptions;
 // Defines
 //
 
-#define NON_BLOCKING_CONNECT true
-#define BLOCKING_CONNECT     false
-#define CONNECT_WITH_TCP     true
-#define CONNECT_WITH_UDP     false
-#define NON_BLOCKING         true
-#define BLOCKING             false
-#define BIND_RANDOM_PORT     true
-#define BIND_ANY_PORT        false
-#define ENABLE_MC_LOOPBACK   true
-#define DISABLE_MC_LOOPBACK  false
-#define BC_NO_CONNECT        true
-#define BC_CONNECT           false
-#define BC_NO_BIND           true
-#define BC_BIND              false
+#define NON_BLOCKING true
+#define BLOCKING     false
```

Comment:
These are only used for NetAccept::do_listen.  Perhaps there is a cleaner way than macros?

Reply:
How about splitting `NetAccept::do_listen` into `NetAccept::do_blocking_listen` and `NetAccept::do_non_blocking_listen` or something similar? The defines are being passed as boolean arguments to the method, so basically someone compensated for `NetAccept::do_listen(false)` being incomprehensible. I quite like how it reads, actually, but I think a more explicit API might be cleaner. I'm worried about the verbosity of splitting it into two methods, though... the macros make it very easy to understand with very little code.

Reply:
It's still suboptimal since `NetAccept::do_listen` called into other APIs that had the same annoyance. I kept it as `NetAccept::do_listen_impl` and wrapped it in `NetAccept::do_listen` and `NetAccept::do_blocking_listen` so that the public API will at least be more presentable.